packages/core/test/unit/query-generator/remove-constraint-query.test.ts

```diff
@@ -0,0 +1,55 @@
+import { createSequelizeInstance, expectsql, sequelize } from '../../support';
+
+const dialect = sequelize.dialect;
+
+describe('QueryGenerator#removeConstraintQuery', () => {
+  const queryGenerator = sequelize.getQueryInterface().queryGenerator;
+
+  it('generates a query that drops a constraint', () => {
+    expectsql(() => queryGenerator.removeConstraintQuery('myTable', 'myConstraint'), {
+      default: 'ALTER TABLE [myTable] DROP CONSTRAINT [myConstraint]',
+    });
+  });
+
+  it('generates a query that drops a column from a model', () => {
+    const MyModel = sequelize.define('myModel', {});
```

Comment:
We should always name the model like it were a class (because it is)

```suggestion
    const MyModel = sequelize.define('MyModel', {});
```

Reply:
Done, also changed in all other instances